.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,22 +4,17 @@
 
 Fixes # (issue)
 
-#### Checklist:
+#### Checklist
 
-- [ ] I have run `./ci/run_stable_checks.sh`
+- [ ] I have run `cargo make pr-flow`
 - [ ] I have reviewed my own code
 - [ ] I have added tests
-<!-- If this is a bug fix, these tests will fail if the bug is present (to stop it from cropping up again) -->
-<!-- If this is a feature, my tests prove that the feature works -->
-
-<!-- Testing instructions -->
-<!-- Check out the link below on how to setup and run tests -->
-<!-- https://github.com/yewstack/yew/blob/master/CONTRIBUTING.md#test -->
-<!-- If you're not sure how to test, let us know and we can provide guidance :) -->
+  <!-- If this is a bug fix, these tests will fail if the bug is present (to stop it from cropping up again) -->
+  <!-- If this is a feature, my tests prove that the feature works -->
 
 <!-- Benchmark instructions -->
+
 <!-- 1. Fork and clone https://github.com/yewstack/js-framework-benchmark -->
 <!-- 2. Update `frameworks/yew/Cargo.toml` with your fork of Yew and the branch for this PR -->
 <!-- 3. Open a new PR with your `Cargo.toml` changes -->
-<!-- 4. Paste a link to the benchmark results: -->
-<!-- - [ ] I have opened a PR against https://github.com/yewstack/js-framework-benchmark -->
+<!-- 4. Include a link to the benchmark results -->
```

Comment:
Should we remove this section entirely? The same (unreliable) benchmarks are already run with the "performance" label anyway.

Reply:
Yeah feel free